lms/djangoapps/completion/models.py

```diff
@@ -105,6 +124,103 @@ def submit_completion(self, user, course_key, block_key, completion):
             )
         return obj, isnew
 
+    def submit_batch_completion(self, user, course_key, blocks):
```

Comment:
This method signature should match `submit_completion`'s, i.e.:
* `user` should be a `User` object
* `course_key` should be a `CourseKey` object
* `blocks` should be a hash of `UsageKey` to `float` values.

Reply:
Ok I'll go ahead and make the changes. When you say hash of UsageKey what do you mean?